media/css/base/banners/vpn-coupon-promo.scss

```diff
@@ -0,0 +1,266 @@
+            background-repeat: no-repeat;
+            background-size: contain;
+            height: 40px;
+            width: 40px;
+            top: 10px;
+            left: 20px;
+        }
+    }
+
+    .c-coupon-content-wrapper {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-around;
+        height: 100%;
+        text-align: center;
+        padding: $spacing-sm;
+
+        // Class for the banner without the promo code title
+        .c-coupon-title {
+            @include text-title-md;
+        }
+
+        .c-coupon-subtitle {
+            @include text-title-sm;
+        }
+
+        .c-coupon-text {
+            font-weight: bold;
+            @include text-body-md;
+            line-height: 1.8;
+
+            .c-coupon-code {
+                border: $border-radius-xs solid $color-white;
+                padding: $border-radius-md $border-radius-lg;
+                margin: 0 $border-radius-sm;
+            }
+        }
+
+        .c-cta {
+            .mzp-c-button {
+                background-color: $color-white;
+                padding-right: $spacing-md;
+                padding-left: $spacing-md;
+
+                &:hover {
+                    background-color: $color-marketing-gray-20;
+                }
+            }
+        }
+    }
+
+    &.c-coupon {
+        .c-banner-close {
+            position: absolute;
+        }
+    }
+
+    // DESKTOP design
+    @media #{$mq-lg} {
+        flex-flow: row nowrap;
+        align-items: center;
+        position: relative;
+        height: 100%;
+
+        .c-coupon-title-wrapper {
+            border-radius: 0;
+            border-bottom-right-radius: $layout-xl;
+            border-top-right-radius: $layout-xl;
+            width: 22%;
+            padding: $spacing-xl;
+
+            .c-coupon-title {
+                @include text-title-xs;
+                text-align: left;
+                padding: 0;
+            }
+
+            // White plane icon
+            &::after {
+                height: 60px;
+                width: 60px;
+                top: 10px;
+                right: -20px;
+                bottom: 0;
+                left: unset;
+            }
+        }
+
+        .c-coupon-content-wrapper {
+            flex-flow: row wrap;
+            align-items: center;
+            justify-content: space-evenly;
+            width: 68%;
+            column-gap: $spacing-lg;
+            row-gap: $spacing-sm;
+
+            .c-coupon-subtitle {
+                @include text-title-2xs;
+
+                .c-coupon-text {
+                    line-height: normal;
+                }
+            }
+        }
+
+        &.c-coupon {
+            .c-close-banner {
+                position: relative;
+            }
+        }
+    }
+
+    // Non-coupon code variation
+    &.c-no-coupon {
+        flex-flow: row nowrap;
+        height: auto;
+        height: $layout-2xl;
```

Comment:
Another fixed height when the content height is probably better. This is 192px which leaves a lot of empty space at medium sizes:
![image](https://user-images.githubusercontent.com/205591/177650539-68118511-58f8-477c-9d73-bf57ed7816c4.png)
Also we should usually avoid using spacing tokens for things that aren't spacing.

Reply:
Should the airplane by on the curved edge or can it float in the middle?